buildbot/master/files/config/factories.py

```diff
@@ -211,10 +213,11 @@ def make_step(self, command):
                 step_kwargs['logfiles'][logfile] = logfile
 
             # Provide environment variables for s3cmd
-            elif arg == './etc/ci/upload_nightly.sh':
+            elif arg == './etc/ci/upload_nightly.sh' or
+                 next(args) == r'.\etc\ci\upload_nightly.sh':
```

Comment:
I was originally using `python .\etc\ci\upload_nightly.sh` in the buildbot_steps.yml file :-) I just found out that it works OK without the python prefix, so I'll remove the next and updated the servo-side PR to the yml file.